packages/compass-import-export/src/hooks/use-toast-action.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { css, Link, spacing } from '@mongodb-js/compass-components';
+
+const toastActionStyles = css({
+  textTransform: 'uppercase',
+  marginLeft: spacing[2],
+});
+
+// Note: This is not in compass-components as having actions in toasts
+// is not recommended by LeafyGreen. We are showing actions in toasts
+// for the time being until we have a dedicated component for background
+// operation actions in Compass.
+export function useToastAction({
+  statusMessage,
+  actionHandler,
+  actionText,
+}: {
+  statusMessage: string;
+  actionHandler: () => void;
+  actionText: string;
+}) {
+  return (
+    <>
+      {statusMessage}
+      <Link
+        data-testid={`toast-action-${actionText}`}
+        onClick={actionHandler}
+        href="#"
```

Comment:
I know that we don't do any a11y anymore, but can we please render this as button instead of a link? It's very low effort to do

Reply:
Good catch, updated